README.md

```diff
@@ -22,9 +22,12 @@ For more information on the supported operations, go to [Using the OpenAPI Tools
 
 ### Setting Up the Prerequisites
 
-1. OpenJDK 11 ([Adopt OpenJDK](https://adoptopenjdk.net/) or any other OpenJDK distribution)
+1. Download and install Java SE Development Kit (JDK) version 17 (from one of the following locations).
 
-    >**Info:** You can also use [Oracle JDK](https://www.oracle.com/java/technologies/javase-downloads.html). Set the JAVA_HOME environment variable to the pathname of the directory into which you installed JDK.
+    * [Oracle](https://www.oracle.com/java/technologies/downloads/)
```

Comment:
Is it okay to suggest oracle distributions in our README files (as it can be misleading for our own internal users within WSO2)? @keizer619 WDYT?

Reply:
Yes Lets suggest Open JDK